roles/nto_run_e2e_test/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Check that nto_git_repo is defined
+  fail: msg="Bailing out. This play requires 'nto_git_repo'"
+  when: nto_git_repo is undefined
+
+- name: Check that nto_git_ref is defined
+  fail: msg="Bailing out. This play requires 'nto_git_ref'"
+  when: nto_git_ref is undefined
+
+- name: Git checkout the NTO repository with e2e tests
+  ansible.builtin.git:
+    repo: "{{ nto_git_repo }}"
+    dest: "{{ nto_tmp_dir }}"
+    version: "{{ nto_git_ref }}"
+    force: true
+
+- name: Store the current git commit
+  shell:
```

Comment:
we need to turn this error into a warning

Reply:
How?

Reply:
And actually, do we even need `git show -s ` ?

Reply:
you can mute it with this task tag (I just discovered it):
```
+  tags: [skip_ansible_lint]
```

> And actually, do we even need git show -s ?

yes, to keep a parsable trace of what is being tested
it's already part of the logs of the task above, but with a dedicated artifact file, it's simpler to confirm what state of the repo is being tested.
eg, the test failed, but I'm sure I fixed it today --> look at the commit, find out if your patch was actually taken into account in the nightly testing